src/pages/test-report/__tests__/test-report.selector.spec.ts

```diff
@@ -18,4 +19,9 @@ describe('TestReportSelectors', () => {
       expect(isDangerousMode(state)).toBeTruthy();
     });
   });
+  describe('isTestValid', () => {
+    it('should return if the test is valid', () => {
```

Comment:
it text should be - should return true if the test is valid

Reply:
Fixed